cdserver/features/shelters/step_definitions/shelter_steps.rb

```diff
@@ -8,9 +8,9 @@
                                                   shelter: shelter)
     floor = Floor.find_or_create_by!(shelter_building: building,
                                      name: entry['Floor'])
-    if entry['Shelter Type'] == 'Single Adult'
-      Bed.create!(name: entry['Unit'], compartment: floor)
-    elsif entry['Shelter Type'] == 'Family'
+    if entry['Population Group'] == 'Single Adult'
```

Comment:
can you make it contains?  That may be better
string contains Family versus contains Single
later we will refactor this for Scheme